mitodl/mitxonline
# feat: Move orders to canceled if transaction is reviewed

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/mitxonline/issues/1404

#### What's this PR do?

- Moves an order to the Canceled state if transaction status by Cybersource is `REVIEW`
- Removes changes made for canceling a reviewed order in the Django admin.

#### How should this be manually tested?
I am not sure how to get a `REVIEW` status from CyberSource but this is what I followed:
- Add a breakpoint in `process_cybersource_payment_response` to modify the Cybersource response.
- Add a product to the cart.
- Place the order.
- Fill in all the payment details.
- Go to the debugger shell and set `processor_response.state = 'REVIEW'` and continue the process.
- You should get an error message that `Payment was canceled.`
- You should also be able to place the order again.
- First order should be in the canceled state.

#### Screenshots (if appropriate)
<img width="1752" alt="Screen Shot 2023-02-07 at 5 55 01 PM" src="https://user-images.githubusercontent.com/52656433/217250801-1d1289cb-493c-400e-826c-4568fa1c723e.png">
